src/app_service/service/community_tokens/service.nim

```diff
@@ -1083,13 +1080,15 @@ QtObject:
     let (ethCurrency, fiatCurrency) = self.create0CurrencyAmounts()
     return ComputeFeeArgs(ethCurrency: ethCurrency, fiatCurrency: fiatCurrency, errorCode: errorCode)
 
+  # Returns eth value with l1 fee included
   proc computeEthValue(self:Service, gasUnits: int, suggestedFees: SuggestedFeesDto): float =
     try:
       let maxFees = suggestedFees.maxFeePerGasM
       let gasPrice = if suggestedFees.eip1559Enabled: maxFees else: suggestedFees.gasPrice
```

Comment:
This line looks misleading. I mean, how `maxFeePerGasM` can be used as `gasPrice`?

Reply:
For EIP-1559 maxFeePerGasM is the absolute maximum you are willing to pay per unit of gas to get your transaction included in a block. You are sure that you will not pay more than that.
So, I understand that you will pay no more than: gas* maxFeePerGasM
For legacy tx you just pay gas*gasPrice.
This is how I understand this.